README.md

```diff
@@ -35,10 +35,10 @@
 - Git & Github
 
 ## Live Demo
-[Live Demo Link]()
+[Live Demo Link](https://berylcl.github.io/HTML-CSS-JavaScript-capstone-project/)
 
 ## Video Walkthrough link
-[Video Walkthrough]()
+[Video Walkthrough](https://www.loom.com/share/6fd228d775e0432fa98cbfcb5b4aa2f7)
 
```

Comment:
- [ ] You did a great job at the presentation of the project and you did 😊⤵️

## Highlights 🚀👌
1. Recorded a video presenting the features of the project.
2. The maximum length of the video is 5 minutes.
3. The video contains a clear and objective description of the project in professional English. The topic and purpose of the project are made clear.


## Forgotten Things 🍀

- [ ] But as stated by the previous reviewer 💯😃 I see that you forgot to 😊

1. The video contains a demo of the project, showing the pages in different screen sizes. 
2. Show some piece of your code as it is from the requirements, so please 😊The video should contain a highlight about a piece of the code the student finds interesting or is proud of making, and with that, your presentation will be great and professional 👌😊